src/Pho/Cli/InitCommand.php

```diff
@@ -77,16 +78,57 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 break;
         }
         
-        $io->title("Project successfully built");
-        $io->text([
+        $this->io->title("Project successfully built");
+        $this->io->text([
             'Emre <href=https://symfony.com>Lorem</> ipsum dolor sit <options=bold,underscore>amet</>',
             'Consectetur adipiscing elit',
             'Aenean sit amet arcu <info>vitae</info> sem faucibus porta',
         ]);
-        //$io->note('Xyz');
-        $io->newLine(1);
-        //$io->success('Lorem ipsum dolor sit amet'); // warning, error
+        //$this->io->note('Xyz');
+        $this->io->newLine(1);
+        //$this->io->success('Lorem ipsum dolor sit amet'); // warning, error
         exit(0);
     }
+    protected function recurseRmdir($dir) {
```

Comment:
let's use Utils::dirDel
this type of functions should go to Utils, especially if it's going to be used by  multiple commands.